docs/faq.rst

```diff
@@ -221,3 +221,8 @@ Why next_ds or prev_ds might not contain expected values?
   ``execution_date`` and ``schedule_interval``. If you set ``schedule_interval`` as ``None`` or ``@once``,
   the ``next_ds``, ``next_ds_nodash``, ``prev_ds``, ``prev_ds_nodash`` valueS will be set to ``None``.
 - When manually triggering DAG, the schedule will be ignored, and ``prev_ds == next_ds == ds``
+
+I am running multiple webserver processes, how can I solve records in the ``ab_permission_view`` with ``null`` ``permission_id``?
+------------------------------------------------
+
+Set the value of ``update_fab_perms`` configuration in ``airflow.cfg`` to ``False`` and have a process that runs the ``sync_perm`` CLI command instead.
```

Comment:
It feels unclear to me on what the issue is from this FAQ question but happy to merge this if @feng-tao thinks so !

Reply:
The null permission ID should have been fixed now via another change anyway, so this is more "how do I stop the sync perms happening multiple times per webserver" etc.

Reply:
#4764 was where the null perm was fixed.